.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+config/prod/* @zaro0508 @xschildw
```

Comment:
required reviews only for files under config/prod folder?  don't we want me and xa to review all files?

Reply:
This is specifically to address the request to create resources. The rest of the code base should be protected as well, but probably separately. The list of people who could approve [config/prod](config/prod/) changes could grow to prevent bottlenecks (e.g., a representative from each team, such as myself and @thomasyu888), plus @brucehoff and others as a fail safe.

Reply:
ahh, ok.  sorry, i thought this was a restriction to only allow me or xa to review which doesn't make much sense.  i've read codeowners doc and get it now.  This sounds very sensible however i think it makes sense to apply reviewers to the entire repo, particularly to the possible file changes in the `templates` folder.